test/test.js

```diff
@@ -54,7 +54,7 @@ const defaultBrowserOptions = {
   args: ['--no-sandbox', '--disable-dev-shm-usage']
 };
 
-const timeout = slowMo ?  0 : 10 * 1000;
+const timeout = slowMo ? 0 : 15 * 1000;
```

Comment:
unfortunately this doesn't help; for some reason appveyour/travis sometimes are very short on resources and take forever to run tests.

Let's split timeout changse out of the PR and discuss separately.

Reply:
Yeah, on my local it was also timing out, after increasing it to 15secs it started working.

Must be the Chromium revision.